Drag.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+import rospy
+from geometry_msgs.msg import WrenchStamped
+from nav_msgs.msg import Odometry
+# Initialized ros parameters and global values (feel free to tweak, values were empirically gained through trial/error)
+rospy.set_param('LINEAR_FORCE', 10)
+rospy.set_param("TORQUE", 5)
+rospy.set_param('ForceDown', -20)
+rospy.set_param('TimeOfForceDown', 10)
+Linear_Drag_X = 0
+Linear_Drag_Y = 0
+Linear_Drag_Z = 0
+Roll_Drag = 0
+Pitch_Drag = 0
+Yaw_Drag = 0
+Velocity = 0
+'''
+Velocities between two time intervals will never be truly 'equal' in real world
+Must compare the abs of the difference of the two with a small delta value
+if the difference is less than this value we can conclude that the numbers are satisfyingly close
+also used as a value that is satisfyingly close to zero. Hard-coded (can be changed).
+'''
+delta = .00001
+# Max velocity cannot be initialized to 0 or there will be an initial divide-by-zero error
+Max_Velocity = .1
+Bouyancy = 0
+# Magnitude of applied force for determining drag coeffcients in linear axes
+appliedLinearForce = rospy.get_param("LINEAR_FORCE", default=10)
```

Comment:
I'd caution against setting defaults that would cause the sub to do something when no config/values are set (in this case I'm assuming there would be some force/torque/etc. applied for some amount of time - 10 seconds?)